CONTRIBUTORS.md

```diff
@@ -11,16 +11,16 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
-    <td align="center"><a href="https://blog.cptjesus.com/"><img src="https://avatars.githubusercontent.com/u/5720446?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Rohan Vazarkar</b></sub></a><br /><a href="https://github.com/BloodHoundAD/AzureHound/commits?author=rvazarkar" title="Code">💻</a> <a href="#maintenance-rvazarkar" title="Maintenance">🚧</a></td>
-    <td align="center"><a href="https://ulises.io/"><img src="https://avatars.githubusercontent.com/u/16910931?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ulises Rangel</b></sub></a><br /><a href="https://github.com/BloodHoundAD/AzureHound/commits?author=urangel" title="Code">💻</a> <a href="#maintenance-urangel" title="Maintenance">🚧</a></td>
-    <td align="center"><a href="https://github.com/joshgantt"><img src="https://avatars.githubusercontent.com/u/29784250?v=4?s=100" width="100px;" alt=""/><br /><sub><b>joshgantt</b></sub></a><br /><a href="https://github.com/BloodHoundAD/AzureHound/commits?author=joshgantt" title="Code">💻</a> <a href="#maintenance-joshgantt" title="Maintenance">🚧</a></td>
-    <td align="center"><a href="https://github.com/hugo-syn"><img src="https://avatars.githubusercontent.com/u/61210734?v=4?s=100" width="100px;" alt=""/><br /><sub><b>hugo-syn</b></sub></a><br /><a href="https://github.com/BloodHoundAD/AzureHound/commits?author=hugo-syn" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/crimike"><img src="https://avatars.githubusercontent.com/u/10261812?v=4?s=100" width="100px;" alt=""/><br /><sub><b>crimike</b></sub></a><br /><a href="https://github.com/BloodHoundAD/AzureHound/commits?author=crimike" title="Code">💻</a></td>
+    <td align="center"><a href="https://www.twitter.com/_wald0"><img src="https://avatars.githubusercontent.com/u/842644?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Andy Robbins</b></sub></a><br /><a href="#ideas-andyrobbins" title="Ideas, Planning, & Feedback">🤔</a> <a href="#design-andyrobbins" title="Design">🎨</a> <a href="#blog-andyrobbins" title="Blogposts">📝</a> <a href="#content-andyrobbins" title="Content">🖋</a> <a href="https://github.com/bloodhoundad/azurehound/v2/commits?author=andyrobbins" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/ddlees"><img src="https://avatars.githubusercontent.com/u/8984872?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dillon Lees</b></sub></a><br /><a href="https://github.com/bloodhoundad/azurehound/v2/commits?author=ddlees" title="Code">💻</a> <a href="#maintenance-ddlees" title="Maintenance">🚧</a> <a href="#ideas-ddlees" title="Ideas, Planning, & Feedback">🤔</a> <a href="#design-ddlees" title="Design">🎨</a></td>
```

Comment:
Looks like you got a little over zealous with the `grep` `sed` replace magic 😉

Reply:
Whoops, thank you for catching that! I have reinstated these